### PR TITLE
Feedback responses

### PR DESCRIPTION
**Pull request by jafcrocker**
This PR reflects responses to Glenn Jones' feedback on the contents of repo.

Each commit represents the response to an individual feedback item.

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -1,8 +1,37 @@
...
+# Several of the targets invoke docker to create a tarball containing the results
+# of some operations (such as installing software).
+#
+# Order-only prerequisites: This is discussed here as many are not familiar with 
+# them. These prerequisites indicate that the prereq should be built if it not 
+# present but that the date on the prereq should not be used to trigger.  
+# Order-only prereqs are used to ensure that a directory is created before any
+# files are placed in it.  All prereqs following the pipe operater (|) in the list
+# of prereqs are order-only.  See 
+#   https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html
```
Comment (tgjonestx) [inline makefile]: 👍

File: makefile [hunk 2]
```diff
@@ -1,32 +1,73 @@
...
+
+
 include versions.mk
 
-ZENPIP := https://zenoss-pip.s3.amazonaws.com/packages
+# Image used for builds in docker containers.
+# Note that this variable is exported for hdfsMetrics recursive make invocations.
+export BUILD_IMAGE := zenoss/build-tools:0.0.2
```
Comment (tgjonestx) [inline makefile]: Please go ahead and bump to 0.0.3 - that will do any java building with Java 8

File: versions.mk [hunk 3]
```diff
@@ -1,17 +1,24 @@
...
-OPENTSDB_IMAGE_VERSION := 24.0.0-dev
+# Versions of third party components used in building the output images
+HBASE_VERSION       := 1.1.4
+OPENTSDB_VERSION    := 2.2.0
+HADOOP_VERSION      := 2.5.2
+ZK_VERSION          := 3.4.5
+HDFSMETRICS_VERSION := 1.0
 
+# Image used as the base from which the output images are built
+BASE_IMAGE       := zenoss/centos-base:1.0.2-java
```
Comment (tgjonestx) [inline versions.mk]: Please update to centos-base:1.1.0-java so that we're using a Java 8 runtime.

**Comment by jafcrocker** [comment]
retest this

**Pull request merged**